### PR TITLE
First releases

### PR DESCRIPTION
**Pull request by FrankGiesecke**

### CODE REVIEW

File: 2.8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM openjdk:8-jdk-alpine3.8
+
+LABEL maintainer="frank.giesecke@final-gene.de"
+
+ENV SONAR_SCANNER_VERSION "2.8"
+ENV SONAR_USER_HOME /cache/.sonar
+ENV SONAR_SCANNER_HOME "/usr/local/bin/sonar-scanner-${SONAR_SCANNER_VERSION}"
```
Comment (MaSpeng) [inline 2.8/Dockerfile]: ❓ SONAR_SCANNER_HOME looks if it is not used

File: README.md [hunk 2]
```diff
@@ -1,2 +1,42 @@
-# docker-hub-sonar-scanner
-Sonar scanner for CI
+# TODO: title
```
Comment (MaSpeng) [inline README.md]: Fix TODO

**Review by MaSpeng — CHANGES_REQUESTED** [review]
With parenthese or without parenthese that is the question 😏 See ENV's in Dockerfiles.

File: 2.8/Dockerfile (hunk 1 above)
Comment (FrankGiesecke) [reply 2.8/Dockerfile]: This is necessary to run the old sonar runner/scanner.

**Review by MaSpeng — APPROVED** [review]

**Pull request merged**